tests/integration/components/model-picker-test.js

```diff
@@ -52,330 +54,318 @@ moduleForComponent('model-picker', 'Integration | Component | {{model-picker}}',
         return [200, { 'Content-Type': 'application/json' }, content];
       });
     });
-  }
-});
+  });
 
-test('Renders with classes and given placeholder', function(assert) {
-  this.render(hbs`{{model-picker model="foo"
-                                 optionLabelPath="name"
-                                 searchProperty="name"
-                                 prompt="Prompt text"
-                                 updated=(action 'update')}}`);
-
-  assert.equal(this.$('.model-picker').length, 1,
-               'renders with model-picker class');
-  assert.equal(this.$('.ember-power-select-placeholder').text().trim(),
-               'Prompt text', 'renders with given prompt as placeholder');
-});
+  hooks.beforeEach(function() {
+    this.actions = {};
+    this.actions.update = function() { };
+    this.send = (actionName, ...args) => this.actions[actionName].apply(this, args);
+  });
 
-test('Searches for given model by attribute', function(assert) {
-  let done = assert.async();
+  test('Renders with classes and given placeholder', async function(assert) {
+    await render(hbs`{{model-picker model="foo"
+                                   optionLabelPath="name"
+                                   searchProperty="name"
+                                   prompt="Prompt text"}}`);
 
-  this.render(hbs`{{model-picker updated=(action 'update')
-                                 model="foo"
-                                 optionLabelPath="name"
-                                 searchProperty="name"}}`);
+    assert.equal(findAll('.model-picker').length, 1,
```

Comment:
Where is this `findAll` coming from?

Reply:
the findAll is part of the ember test helpers